app/assets/json/translations/locale-es.json

```diff
@@ -470,7 +470,6 @@
                 "lottery_will_be_held": "El sorteo se llevará a cabo el %{lottery_date}. Los resultados del sorteo de viviendas se publicarán en %{link}.",
                 "office_hours": "Horarios de atención",
                 "on_the_listing": "en el listado",
-                "notify_about_potential_resend": "<b>Si ya recibió este correo electrónico de confirmación de nuestra parte, ignore este mensaje.</b> Tuvimos un problema al enviar correos electrónicos desde DAHLIA recientemente, por lo que los enviamos nuevamente. Hicimos esto para asegurarnos de que todos los solicitantes tengan sus números de lotería.",
```

Comment:
@tallulahkay Something is wrong with our Angular translations in phrase, probably related to some work I did for the spam incident. I'm looking into this now.

Reply:
@tallulahkay I've updated Angular translations in phrase. Running `grunt phrasePull` again should fix this issue.